definitions/procedures/foreman_tasks_delete/aged.rb

```diff
@@ -0,0 +1,18 @@
+module Procedures::ForemanTasksDelete
+  class Aged < ForemanMaintain::Procedure
+    label :delete_old_foreman_tasks
+    description 'Delete Old Tasks'
+
+    def run
+      old_tasks_count = feature(:foreman_tasks).old_tasks_count
```

Comment:
Once we get https://github.com/iNecas/foreman_maintain/pull/29 in, it should be possible to parametrize the procedure, where we could specify the parameters of which kind of tasks we want to delete.

What would be a reason to delete one kind of tasks?